inc/saiport.h

```diff
@@ -431,6 +433,14 @@ typedef enum _sai_port_attr_t
      * given port list will be dropped. */
     SAI_PORT_ATTR_EGRESS_BLOCK_PORT_LIST,
 
+    /** Dropping of 802.1BR untagged frames on ingress [bool] (default to FALSE).
+     * Applicable only to Physical ports. */
+    SAI_PORT_ATTR_DOT1BR_DROP_UNTAGGED,
```

Comment:
How these attributes interact with dot1br and dot1br_extended port objects. If a given port is set as physical port for one of the br types (cascade, upstream, access) , we can use the br type as a hint to derive drop etag tagged/untagged. Is there some other use case I am missing?